tests/fixtures/merge-ntfs/output/report.json

```diff
@@ -1 +1 @@
-{"errors":[],"warnings":[{"category":"TransferOnNonExistentStop","message":"manual transfer references non-existent stop points (101938 and 101939)"},{"category":"TransferOnNonExistentStop","message":"manual transfer references non-existent stop points (UNKNOWN and 101938)"},{"category":"TransferOnNonExistentStop","message":"manual transfer references non-existent stop points (101938 and UNKNOWN)"},{"category":"TransferOnNonExistentStop","message":"manual transfer references non-existent stop points (101941 and 101938)"},{"category":"TransferOnNonExistentStop","message":"manual transfer references non-existent stop points (101937 and 101937)"},{"category":"TransferOnNonExistentStop","message":"manual transfer references an non-existent stop point (DEF)"},{"category":"TransferIntraIgnored","message":"transfer between stops OIF:SP:10:100 and OIF:SP:10:10 is ignored"},{"category":"TransferOnNonExistentStop","message":"manual transfer references an non-existent stop point (DEF)"},{"category":"TransferOnUnreferencedStop","message":"stop point OIF:SP:10:300 belongs to none of the trips and will not generate any transfer"},{"category":"TransferIntraIgnored","message":"transfer between stops CDGR and OIF:SP:10:300 is ignored"}]}
```

Comment:
Just to verify I understand why this file changed, we don't write messages in duplicate anymore, right?

Reply:
Yes. ```{"category":"TransferOnNonExistentStop","message":"manual transfer references an non-existent stop point (DEF)"}``` was duplicated.